src/Tools/Microsoft.dotnet-openapi/src/CodeGenerator.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.DotNet.OpenApi
+{
+    public enum CodeGenerator
+    {
+        NSwagCSharp
```

Comment:
Doesn't include `NSwagTypeScript` because I was never told that that was a thing. Let's chat offline about that the gap is here and how we can close it while preferably keeping this in preview9.